examples/ex_cpu.c

```diff
@@ -0,0 +1,92 @@
+/* An example showing the use of al_get_cpu_count() and al_get_ram_size(). */
```

Comment:
Sure, but for an example app end user it's easier to see the results on the screen of their phone than to have to look into the adb debug logs. So I'd like to leave ex_cpu as an app with a display until the native dialog on androids work well enough to display the log on the phone.